src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTreeNodes.cs

```diff
@@ -213,6 +213,11 @@ public override IEnumerable<NodeSmoProperty> SmoProperties
                    Name = "Status",
                    ValidFor = ValidForFlag.All
                 });
+                properties.Add(new NodeSmoProperty
+                {
+                   Name = "IsLedger",
```

Comment:
>IsLedger", [](http://example.com/codeflow?start=27&length=10)

nameof(Database.IsLedger) and nameof(Database.Status) above

Reply:
This file is auto-generated. You'd need to update the code gen to make this change.